drivers/flash/flash_simulator_stub/Kconfig

```diff
@@ -0,0 +1,25 @@
+# Flash simulator config
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig FLASH_SIMULATOR_STUB
```

Comment:
following the comment here https://github.com/nrfconnect/sdk-nrf/pull/5620#issuecomment-919983174
I think `FLASH_RAM` might be a better naming.

Reply:
Isn't it the stub?

Reply:
I was referring to our talks regarding the simulator naming not being optimal name for NCS users.
The name simulator might be confusing to them when the look at this.

So why not choose a better name now, and not having to rename later ?

Remember, simulator will pop up in Kconfig, conf files, file names in IDEs, etc.

Reply:
The name has been chosen to "FLASH_NOP_DEVICE"